faq/tutum-on-packet.net-faq.md

```diff
@@ -0,0 +1,29 @@
+## Which objects does Tutum create in my Packet.net account?
+
+Tutum will create a project named "Tutum" which will contain all the devices that Tutum deploys, no matter what type of device you chose.
+
+Device storage is organized the following way:
+
+- Type 1 devices has a RAID 1 of two SSD drives mounted in "/"
+- Type 3 devices also has a RAID 1 of two SSD drives mounted in "/", and also offers two NVMe drives without being mounted. Tutum mounts a RAID 1 in "/var/lib/docker".
+
+An SSH keypair named `tutum-<uuid>` will be created if no key is found in your account.
+
+## How long does it take to deploy a Packet.net device?
+
+Tutum deploys Ubuntu 14.04 LTS images on both types. Type 1 takes up to 5 ~ 10 min to be ready, while type 3 takes 8 ~ 13 min. Packet.net guys are working hard to reduce this deployment times.
+
+## What happens if I restart a node in the Packet.net portal?
+
+After the node boots up, Tutum Agent will try contact our API and register itself with the new IP. We will update the DNS of the node and the containers on it to use the new IP automatically. The node should change its state from `Unreachable` to `Deployed`.
+
+## Can I terminate a node from the Packet.net portal?
+
+If you created the node using Tutum and you terminate it in the portal, all data in that node will be destroyed. We will detect the termination and will mark the node as `Terminated` on Tutum.
+If you turn off the device, Tutum will mark it as `Unreachable` because the node is not terminated but we cannot operate with it. Other possible states will be considered as `Terminated`.
```

Comment:
```
Other possible states will be considered as `Terminated`.
```

I hope this is not true? We only consider nodes as terminated if they are actually marked as terminated in the provider.